### DISCUSSION TITLE
What will be the value of below Environment variable for FEG

### DISCUSSION THREAD

**Discussion by vik2595** [magma/magma#7516] [Q&A] [answered]
Hi All, 

I'm missing following environment variable while installing FEG

WARNING: 
```
WARNING: The LOG_DRIVER variable is not set. Defaulting to a blank string.
WARNING: The USE_GY_FOR_AUTH_ONLY variable is not set. Defaulting to a blank string.
WARNING: The GY_SUPPORTED_VENDOR_IDS variable is not set. Defaulting to a blank string.
```

What could be the value of following environment variable ? 
```
USE_GY_FOR_AUTH_ONLY
GY_SUPPORTED_VENDOR_IDS
GY_SERVICE_CONTEXT_ID
```
Please advice @arunuke @karthiksubraveti

**Comment by sourabh-nanoti** [comment, marked as answer]
Hey @vik2595 - 
USE_GY_FOR_AUTH_ONLY -  This  was added for a specific use case where where during attach, the GyCCR-I/CCA-I still had to be done even if there were no GY tracked rules. The use case was that subscriber barring was determined via Gy CCA-I. If CCA-I resulted in a specifc result code, magma could identify that the subscriber is barred. 
GY_SUPPORTED_VENDOR_IDS and GY_SERVICE_CONTEXT_ID were added to fill in any additional vendor IDs or to add any custom service context id to interop with specific OCS. 
The idea was to expose these via NMS if these are needed quite often in deployments. However that does not seem to be the case yet.
You can safely ignore these warnings for now and i will raise  a PR to suppress these in coming releases.

**Reply by pramchan** [reply]
LGTM
